experimental/apps-mcp/lib/prompts/apps.tmpl

```diff
@@ -9,6 +9,21 @@
 DATABRICKS APPS DEVELOPMENT
 ============================
 
+When building or planning a new Databricks app, use the following steps:
+
+1. MUST start by scaffolding a new Databricks app using command: invoke_databricks_cli 'experimental apps-mcp tools init-template --name my-app-name --description "My app description"'
+   MUST run this command before planning the app. You must understand tge generated file structure and the purpose of the files.
+2. MUST read the CLAUDE.md file in the app directory.
+3. SHOULD run `npm install`, run `npm run dev` in the background, and open http://localhost:8000 in your browser before making changes to the app.
```

Comment:
why? I am under impression we try to encapsulate lower level ops with validation step